docs/src/ci.md

```diff
@@ -587,6 +587,20 @@ tests:
     - npm ci
     - npx playwright test --project=$PROJECT --shard=$SHARD
 ```
+### Google Cloud Build
+
+To run Playwright tests on Google Cloud Build, use our public Docker image ([see Dockerfile](./docker.md)).
+
+```yml
+steps:
+- name: 'mcr.microsoft.com/playwright:v%%VERSION%%-jammy'
```

Comment:
we don't use quotes around image name in other places, let's be consistent

Reply:
Done